integration-tests/plugins/medusa-config.js

```diff
@@ -37,13 +36,7 @@ module.exports = {
     database_extra: { idle_in_transaction_session_timeout: 0 },
   },
   featureFlags: {
-    isolate_product_domain: enableProduct,
-    isolate_pricing_domain: enablePricing,
-    workflows: {
-      [Workflows.CreateProducts]: true,
-      [Workflows.UpdateProducts]: true,
-      [Workflows.CreateCart]: true,
```

Comment:
comment: Workflows will be able to work without isolated modules, so maybe we should actually keep this.

My thinking is if we open for workflow extensibility before we release Medusa V2, it would be nice for users to hook into flows like add to cart or create cart without having to also opt-in for experimental modules work. 

Create cart and add to cart don't require isolated modules.

What do you think?

Reply:
You mean, just allow enabling it for cart workflows and not product workflows? That could work since pricing is not touching that at the moment.

But if you mean enabling it for products as well, we currently don't have a code path in the workflows and endpoints without pricing module as we had agreed on earlier 😅

Reply:
Yes, I would only use it to enable workflows, that don't use modules. So not products.

Reply:
@olivermrbl readded the create cart workflow FF